capz/run-capz-e2e.sh

```diff
@@ -105,6 +108,8 @@ create_cluster(){
         template="$SCRIPT_ROOT"/templates/windows-base.yaml
         if [[ "${GMSA}" == "true" ]]; then
             template="$SCRIPT_ROOT"/templates/gmsa.yaml
+        elif [[ "${PRIVATE_TESTING}" == "true" ]]; then
+            template="$SCRIPT_ROOT"/templates/private-test.yaml
```

Comment:
do you want to / need to run GMSA tests when `PRIVATE_TESTING` is true?

Reply:
I don't need it.